SPECS/SymCrypt/SymCrypt.spec

```diff
@@ -40,15 +40,20 @@ A core cryptographic library written by Microsoft
 %setup -q
 %setup -q -a 1
 # Create a symbolic link as if jitterentropy-library has been pulled in as git submodule
-rm -rf jitterentropy-library
-ln -s jitterentropy-library-3.3.1 jitterentropy-library
+rm -rf 3rdparty/jitterentropy-library
+ln -s ../jitterentropy-library-3.3.1 3rdparty/jitterentropy-library
 
 %build
+SYMCRYPT_BRANCH=main \
+SYMCRYPT_COMMIT_HASH=c55c670 \
+SYMCRYPT_COMMIT_TIMESTAMP=2023-12-01T15:59:37-08:00 \
```

Comment:
I think the hash and timestamp should reflect the hash and timestamp of the last commit in SymCrypt 103.4.1 just for clarity when these strings are built into the library:
SYMCRYPT_COMMIT_HASH=a84ffe1
SYMCRYPT_COMMIT_TIMESTAMP=2024-01-26T22:00:47-08:00

Reply:
Thanks for the catch. I forgot to update this after the release was published.